GTG/gtk/editor/taskview.py

```diff
@@ -463,6 +463,37 @@ def insert_tag(self, tag):
         line_mark = self.buff.create_mark("lastline", lastline, False)
         self.insert_at_mark(self.buff, line_mark, tag)
 
+    def remove_tag_editor(self, tagname):
+        '''
+        TODO(jakubbrindza): Finish works on this function which will
+        enable removing a tag from a task in task editor.
```

Comment:
but then the entire functionality of tags will not be complete. Is that an acceptable way to handle this? I think it will only raise tons of complaints and bugs from the users.

However if you think that this is extra feature (which it is) and would deserve a separate PR then I cannot oppose and I remove the entire body of the function for now :blush: